swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+import copy
+
+# Wave 1 
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=[]):
+        self.inventory = copy.copy(inventory)
+
+    def add(self, item): 
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory: 
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
```

Comment:
Because the `if` block returns, we don't need the `else`. we could just write:

```
    def remove(self, item):
        if item not in self.inventory: 
            return False
        self.inventory.remove(item)
        return item
```